ampersand-dom-bindings.js

```diff
@@ -5,13 +5,18 @@ var matchesSelector = require('matches-selector');
 var partial = require('lodash.partial');
 var slice = Array.prototype.slice;
 
-function getMatches(el, selector) {
+function getMatches(el, selector, firstOnly) {
     if (selector === '') return [el];
     var matches = [];
-    if (matchesSelector(el, selector)) matches.push(el);
-    return matches.concat(slice.call(el.querySelectorAll(selector)));
+    if(!selector) return matches;
```

Comment:
Yep, but unless we get really picky I wouldn't remove it. Check on _switch_, _toggle_, and cases with yes/no values, the selector is ignored in favor of yes/no/cases props which are not passed through getSelector. I didn't do the full trace, but that little if can save your debugging life. Maybe some console message to give feedback?

Anyways, I'd leave it there, doesn't hurt much. What do you think?